src/context/mapArea.provider.tsx

```diff
@@ -51,7 +70,17 @@ export const MapAreaProvider: React.FC<{ children: ReactNode }> = ({ children })
     isOpen: false,
     uuid: ""
   });
-
+  const [polygonNotificationStatus, setpolygonNotificationStatus] = useState<{
+    open: boolean;
+    message: string;
+    type: "success" | "error" | "warning";
+    title: string;
+  }>({
+    open: false,
+    message: "",
+    type: "success",
+    title: "Success!"
```

Comment:
Needs translation

Reply:
removed because is init state and shouldnt have title.